app/controllers/api/v0/base_controller.rb

```diff
@@ -0,0 +1,37 @@
+class Api::V0::BaseController < ApplicationController
+  before_action :verify_client_name_and_signature
+  rescue_from MefService::RetryableError, with: :retryable_mef_error
+  rescue_from ActionController::ParameterMissing, with: :show_errors
+  rescue_from Aws::SecretsManager::Errors::ResourceNotFoundException, with: :show_errors
+  rescue_from JWT::VerificationError, with: :show_errors
+
+  def retryable_mef_error
+    render json: "Error contacting MeF, please try again"
+  end
```

Comment:
dumb question, what http status will this be...?

Reply:
Good catch - the `rescue_from` handlers all now attach a reasonable http status